### PR TITLE
Create RUNBOOK.md

### PR DESCRIPTION
**Pull request by 10thfloor**
Closes: #209

### CODE REVIEW

File: RUNBOOK.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+2) Push your tag to the main branch: `git push origin --tags`
+3) To trigger a staging deployment, update the version number here: https://github.com/dapperlabs/flow-playground/blob/master/Makefile#L3
+    - Make sure the version number here, matches your new tag, if you want your changes to be deployed. 
+4) Once updated visit: https://ci.eng.dapperlabs.com/buildConfiguration/Flow_FlowPlayground_FlowTestNetwork_BuildCiCdPlaygroundStaging#all-projects
+    - Here, you will see your deployment in the queue, and the build log output.
+5) If the Teamcity build succeeds your chnges will be available to preview here: https://play.staging.onflow.org
+
+**Production Deployment**
+
+// TODO
```
Comment (kerrywei) [inline RUNBOOK.md:58]: to be finished?

File: RUNBOOK.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+### Access
+
+To create a Playground deployment, your user must: 
+- Be added to the Flow/Dapper Labs Teamcity account
+- Have write access to this repository for triggering Teamcity deploys (details below): https://github.com/dapperlabs/flow-playground
+- Be connected to the Office VPN
+
+A request for these should be filed here: https://dapperlabs.happyfox.com/
+
+### Deployment Workflow
```
Comment (kerrywei) [inline RUNBOOK.md:42]: would we want to link to the internal deployment doc? recommend not duplicate contents but aggregate them into one place as one single source of trueth

**Review by kerrywei — COMMENTED** [review]
nice work Mac!

**Review by kerrywei — APPROVED** [review]

File: RUNBOOK.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+2) Push your tag to the main branch: `git push origin --tags`
+3) To trigger a staging deployment, update the version number here: https://github.com/dapperlabs/flow-playground/blob/master/Makefile#L3
+    - Make sure the version number here, matches your new tag, if you want your changes to be deployed. 
+4) Once updated visit: https://ci.eng.dapperlabs.com/buildConfiguration/Flow_FlowPlayground_FlowTestNetwork_BuildCiCdPlaygroundStaging#all-projects
+    - Here, you will see your deployment in the queue, and the build log output.
+5) If the Teamcity build succeeds your chnges will be available to preview here: https://play.staging.onflow.org
+
+**Production Deployment**
+
+Once a staging deployment has been verified. To move the changes into produciton, navigate to the production pipeline in Teamcity here: https://ci.eng.dapperlabs.com/buildConfiguration/Flow_FlowPlayground_FlowTestNetwork_BuildCiCdPlaygroundProduction#all-projects and click the "Deploy" button in the top right of the table.
```
Comment (MaxStalker) [inline RUNBOOK.md:58]:
```suggestion
Once a staging deployment has been verified. To move the changes into produciton, navigate to the production pipeline in TeamCity here: https://ci.eng.dapperlabs.com/buildConfiguration/Flow_FlowPlayground_FlowTestNetwork_BuildCiCdPlaygroundProduction#all-projects and click the "Deploy" button in the top right of the table. 
```

**Review by MaxStalker — APPROVED** [review]
Great work, @10thfloor 
Left a couple suggestions. Mainly about `TeamCity` name :sweat_smile:

File: RUNBOOK.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+1. Open a new pull request and select `production` as the base branch and `staging` as the source branch. Production deployments should always deploy the same code that is already live in staging. Vercel will trigger a new deployment once the PR is approved and merged.
+
+2. Vercel will then report the deployment status on the `production` branch:
+
+![vercel-deployment](vercel-deployment.png)
+
+## Additional Details
+
+More details and screenshots regarding the deployment process [can be found on Notion](
+https://www.notion.so/dapperlabs/Flow-Playground-Deployment-Process-6ca452adb63a4b41bbe5f3b56eda7021).
```
Comment (kerrywei) [inline RUNBOOK.md:68]: is this doc still needed?

File: RUNBOOK.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+### Monaco Editor
+
+- The editor interface itself is implemented using [Monaco Editor](https://microsoft.github.io/monaco-editor/).
+- The editor component can be found here: https://github.com/onflow/flow-playground/tree/master/src/containers/Editor
+- The Cadence language definition (for linting and syntax highlighting) for Monaco can be found here: https://github.com/onflow/flow-playground/blob/master/src/util/cadence.ts
+
+### Cadence Language Server
+
+- The Cadence Language Server (protocol for Monaco) is implemented in Goland and compiled to WASM.
```
Comment (kerrywei) [inline RUNBOOK.md:28]: "(protocol for Monaco)" -- is this intended to explain language server?

**Review by kerrywei — COMMENTED** [review]
no other comments. thank you for the great work @psiemens ❤️

File: RUNBOOK.md (hunk 4 above)
Comment (psiemens) [reply RUNBOOK.md:68]: No it's not actually. Probably best not to include links to private pages anyways. Thanks!

File: RUNBOOK.md (hunk 5 above)
Comment (psiemens) [reply RUNBOOK.md:28]: Yes, but adjusted the wording a bit

**Pull request merged**